proposals/implicit-lambda-parameters-with-modifiers.md

```diff
@@ -1,10 +1,10 @@
-# Declaration of lambda parameters with modifiers without type name
+# Implicit lambda parameters with modifiers
```

Comment:
Can't say I like this rename. Perhaps `implicitly-typed`? The parameter isn't implicit.

Reply:
The spec calls them "implicit parameters" versus "explicit parameters"- but I don't disagree. What if I drop the commit so that I call them "simple parameters" instead?

Reply:
That works for me.